src/App.js

```diff
@@ -1,9 +1,16 @@
 import './App.css';
+import Header from './Components/Header';
+import Card from './Components/Card'
+import Footer from './Components/Footer';
 
 function App() {
   return (
     <div className="App">
-     
+     <Header/>
+     <main>
+       <Card></Card>
```

Comment:
You can use a self closing component here
```suggestion
       <Card />
```